src/HyperspaceCloud.cpp

```diff
@@ -44,6 +44,7 @@ HyperspaceCloud::HyperspaceCloud(const Json &jsonObj, Space *space) :
 		m_due = hyperspaceCloudObj["due"];
 		m_isArrival = hyperspaceCloudObj["is_arrival"];
 
+		m_ship = 0;
```

Comment:
This should be `nullptr` rather than `0` which we're trying to discourage usage of.